readme.md

```diff
@@ -153,18 +154,19 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. F
 
 #### Other apps
 
-*Apps that support WebDAV in some form, e.g. for backup and sync*
+_Apps that support WebDAV in some form, e.g. for backup and sync_
 
 - [1Writer](https://apps.apple.com/app/1writer-markdown-text-editor/id680469088) - markdown text editor that supports importing from WebDAV
 - [beorg](https://apps.apple.com/app/beorg-to-do-list-agenda/id1238649962) - to-do list and agenda app with WebDAV sync support
 - [GoodReader](https://apps.apple.com/app/goodreader-pdf-editor-viewer/id777310222) - PDF viewer and editor that supports WebDAV sync
 - [Joplin](https://apps.apple.com/app/joplin/id1315599797) - note taking and to-do application that supports WebDAV sync
 - [Notebooks 10](https://apps.apple.com/app/notebooks-write-and-organize/id1490084838) - writing app, a text and markdown editor with WebDAV support
 - [Strongbox](https://apps.apple.com/app/strongbox-password-safe/id897283731) - KeePass-based password manager that supports WebDAV sync
+- [Notability](https://apps.apple.com/us/app/notability/id360593530) - Note-taking app that supports webDAV for backups
```

Comment:
Could you remove the `/us` from the URL to prevent a redirect, i.e. `https://apps.apple.com/app/notability/id360593530`?

Reply:
Done!